### PR TITLE
Update Neo nuget

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo-devpack-dotnet/issues/322
Require https://github.com/neo-project/neo-devpack-dotnet/pull/324

### CODE REVIEW

**Comment by ProDog** [comment]
In neo-core commits, https://github.com/neo-project/neo/pull/1751 is before https://github.com/neo-project/neo/pull/1729 😀

File: src/Neo.Compiler.MSIL/Program.cs [hunk 1]
```diff
@@ -334,7 +334,7 @@ public static int Compile(Options options)
             return -1;
         }
 
-        private static object BuildSupportedStandards(Mono.Collections.Generic.Collection<CustomAttributeArgument> supportedStandardsAttribute)
+        internal static object BuildSupportedStandards(Mono.Collections.Generic.Collection<CustomAttributeArgument> supportedStandardsAttribute)
```
Comment (erikzhang) [inline src/Neo.Compiler.MSIL/Program.cs:337]: Why `internal`?

File: src/Neo.Compiler.MSIL/Program.cs [hunk 2]
```diff
@@ -353,7 +353,7 @@ private static object BuildSupportedStandards(Mono.Collections.Generic.Collectio
             return extra;
         }
 
-        private static string BuildExtraAttributes(List<Mono.Collections.Generic.Collection<CustomAttributeArgument>> extraAttributes)
+        internal static string BuildExtraAttributes(List<Mono.Collections.Generic.Collection<CustomAttributeArgument>> extraAttributes)
```
Comment (erikzhang) [inline src/Neo.Compiler.MSIL/Program.cs:356]: Why `internal`?

Comment (shargon) [reply src/Neo.Compiler.MSIL/Program.cs:356]: In unit tests we duplicate this part, so I reused the method. https://github.com/neo-project/neo-devpack-dotnet/pull/325/files#diff-9be8e240c7558058b8d39e89572dcd15R109

Comment (shargon) [reply src/Neo.Compiler.MSIL/Program.cs]: I changed the code to unify the manifest generation here https://github.com/neo-project/neo-devpack-dotnet/pull/325/commits/5e07dd0fbcc01612a309b207ca0a0c88661abb8c

**Pull request merged**